dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -252,6 +252,114 @@ html, body {
   }
 }
 
+/* Styling that keeps the small footer consistent with other page elements */
+$codeApp-left-margin: 25px;
+$small-footer-horizontal-margin: 15px;
+$small-footer-standard-width: 400px;
+
+/* Shared styles so flyout looks like footer */
+.small-footer, #copyright-flyout, #more-menu {
+  background-color: #ddd; /* Matches blockly flyout */
+  color: $light_gray;
+  font-size: 14px;
+  a {
+    &:link {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color: $light_gray;
+      text-decoration: none;
+    }
+    &:visited {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color: $light_gray;
+      text-decoration: none;
+    }
+    &:hover {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color:$dark_charcoal;
+      background-color: transparent;
+      text-decoration: underline;
+    }
+    &:active {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color:$light_gray;
+      background-color: transparent;
+      text-decoration: none;
+    }
+  }
+}
+
+.small-footer {
+  position: absolute;
+  z-index: 3;
+  bottom: 0;
+  width: $small-footer-standard-width;
+  overflow: hidden;
+  padding-left: $codeApp-left-margin;
+  padding-top: 3px;
+  padding-bottom: 3px;
+  margin-top: 5px; /* Used in javascript to position #belowVisualization */
+
+  form {
+    float: left;
+    margin-right: $small-footer-horizontal-margin;
+  }
+
+  select {
+    width: 80px;
+    font-size:11px;
+    height: 18px;
+    margin-bottom: 0;
+  }
+
+  small {
+    display: block;
+    float: right;
+    margin-right: $small-footer-horizontal-margin;
+    padding-top: 3px;
+    white-space: nowrap;
+  }
+}
+
+#copyright-flyout {
+  display: none; /* Invisible until summoned. */
+  position: absolute;
+  z-index: 2;
+  bottom: 0;
+  left: 0;
+  width: 625px;
+  max-width: 50%;
+  min-width: 300px;
+  padding: 1em;
+}
+
+#more-menu {
+  display: none;
+  position: absolute;
+  z-index: 1;
```

Comment:
This may be a silly thought, and not sure if there's an idiomatic way of doing this—

Since the relative order of these is important, wonder if it might make sense to define these in a clump above the selector style definitions, i.e.,

```
$more-z-index: 1;
$copyright-z-index: 2;
$footer-z-index: 3;

.small-footer {
  [...]
```

or maybe re-open the definitions in a clump

```
#more-menu { z-index: 1; }
#copyright-flyout { z-index: 2; }
.small-footer { z-index: 3; } 
```